src/index.html

```diff
@@ -1,17 +1,245 @@
 <!doctype html>
 <html lang="en">
+
   <head>
-    <meta charset="UTF-8">
+    <meta
+      charset="UTF-8"
+    >
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
```

Comment:
Dont spread attribute value, put it in a one line
```suggestion
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
```